taskassigned/crud_without_serializers/views.py

```diff
@@ -23,25 +22,31 @@ def get(self,request):
 
 
     def post(self,request):
```

Comment:
check if same record already exists . will not create record again or update

Reply:
ok will check it